src/Restivus/Extensions.cs

```diff
@@ -73,15 +81,27 @@ public static HttpRequestMessage SetQueryParams(this HttpRequestMessage request,
             return request.UpdateRequestUri(uri => uri.SetQueryParams(queryParams));
         }
 
+#if NETSTANDARD13
+        public static Uri UpdateQueryParams(this Uri uri, Func<Dictionary<string, StringValues>, Dictionary<string, StringValues>> updateFn)
+#elif NET4_5
         public static Uri UpdateQueryParams(this Uri uri, Func<NameValueCollection, NameValueCollection> updateFn)
+#endif
         {
             return new UriBuilder(uri)
             {
+#if NETSTANDARD13
+                Query = updateFn(QueryHelpers.ParseQuery(uri.Query)).ToString(),
+#elif NET4_5
                 Query = updateFn(HttpUtility.ParseQueryString(uri.Query)).ToString(),
+#endif
             }.Uri;
         }
```

Comment:
Since `UpdateQueryParams` is so simple, I think it'd be simpler / more readable to just have separate overloads each contained within their own branches of this conditional, rather than cutting it up this way.

```cs
#if NETSTANDARD13
    public static Uri UpdateQueryParams(this Uri uri, Func<Dictionary...)
    {
        // ...
    }
#elif NET4_5
    public static Uri UpdateQueryParams(this Uri uri, Func<NameValueCollection...)
    {
        // ...
    }
#end
```